lib/src/utils/currency_utils.dart

```diff
@@ -116,6 +166,14 @@ class CurrencyUtil {
           final calculated = number / 1000;
           return '${formatter.format(calculated)} K';
         }
+      } else if (companyUsesFrenchForCurrencyConversion) {
```

Comment:
duplicate code, similar code already added in above function please resuse if possible.